modules/custom/az_event/az_event_trellis/config/install/migrate_plus.migration.az_trellis_events.yml

```diff
@@ -50,10 +50,33 @@ source:
       name: publish
       label: Publish
       selector: Publish_Formula
+    -
+      name: az_enterprise_attributes
+      label: 'Enterprise Attributes'
+      selector: Attributes
   ids:
     id:
       type: string
 process:
+  pseudo_az_enterprise_attributes:
+    -
+      plugin: az_flatten_enterprise_attributes_array
+      source: az_enterprise_attributes
+    -
+      plugin: skip_on_empty
+      method: process
+    -
+      plugin: flatten
```

Comment:
Having to flatten after running through `az_flatten_enterprise_attributes_array` seems weird to me.